cairo/src/utils/bytes.cairo

```diff
@@ -429,3 +431,37 @@ func keccak{range_check_ptr, bitwise_ptr: BitwiseBuiltin*, keccak_ptr: KeccakBui
     let (code_hash) = keccak_bigend(dst, code_len);
     return code_hash;
 }
+
+// @notice Converts a sequence of bytes to a Uint256 value (little endian)
+// @param len: number of bytes
+// @param ptr: pointer to bytes array
+// @return: Uint256 value
+func bytes_be_to_uint256{range_check_ptr}(len: felt, ptr: felt*) -> Uint256 {
```

Comment:
i don't get it: what is `be`? what is `le`?

Reply:
I think a more logical name would be `uint256_from_bytes_be` if I understood well

Reply:
ok! it's simply that we have bytes_to_felt in the codebase